### PR TITLE
removed unused imports

### PR DESCRIPTION
**Pull request by sergiy-petrov**
I've removed all unused imports in project.

### CODE REVIEW

File: Slim/Route.php [hunk 1]
```diff
@@ -8,8 +8,6 @@
  */
 namespace Slim;
 
-use Exception;
```
Comment (damianopetrungaro) [inline Slim/Route.php:11]: This is used in https://github.com/slimphp/Slim/pull/2395/files#diff-5cbcc564b9520431aac614f483da529cR312
But use the backslash, so you may remove it or use the backslash here.

Comment (akrabat) [reply Slim/Route.php:11]: Please do not use `\Exception` inline. import it with `use`.

**Comment by akrabat** [comment]
Needs rebasing and removal of the use of `\Exception`

**Comment by damianopetrungaro** [comment]
@akrabat btw we may add a cs fixer and automate all those processes.

**Comment by akrabat** [comment]
> @akrabat btw we may add a cs fixer and automate all those processes.

Good idea.

**Comment by damianopetrungaro** [comment]
I can work on it tomorrow if is it ok for you.

**Comment by akrabat** [comment]
Closing as requested changes haven't been completed and we may go a different route.

**Pull request closed without merging**